src/styles/FAQ.module.less

```diff
@@ -0,0 +1,3 @@
+.FAQ {
+  padding: 0.5rem 10rem;
+}
```

Comment:
Is there a difference in `.less` vs `.module.less` files?

Reply:
None, will move css into `.module.less` to `.less`